bpdv3/Assets/Scripts/Floor/TileController.cs

```diff
@@ -25,15 +25,17 @@ void Start()
     // Update is called once per frame
     void Update()
     {
+        // Specified by the GDD, on the 6th beat the floor moves until it reaches the 12th beat. Seen below **.
         if (SceneController.instance.scrollBeatCount >= 6)
         {
             foreach (GameObject go in tilesArray)
```

Comment:
Please do not use short version of any vairable, it reduces readability too much. go => gameObject